storybook/cypress/e2e/responsive-element.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe("Responsive element", () => {
+  const responsiveElementSelector = "dso-responsive-element.hydrated";
+  beforeEach(() => {
+    cy.visit("http://localhost:45000/iframe.html?id=core-responsive-element--responsive-element");
+  });
+
+  it("should be accessible - default", () => {
+    cy.injectAxe();
+    cy.dsoCheckA11y(responsiveElementSelector);
+  });
+
+  it("should have size (small, medium, large) attribute depending on the viewport", () => {
+    cy.viewport(1312, 660);
+    cy.get(responsiveElementSelector).first().should("have.attr", "large");
+    cy.matchImageSnapshot();
+
+    cy.viewport(992, 660);
+    cy.get(responsiveElementSelector).first().should("have.attr", "medium");
+    cy.matchImageSnapshot();
```

Comment:
Omdat je er geen naam aan meegeeft gaat het mis bij deze. 
Want de eerste krijgt de naam van de test en de tweede vervolgens ook en die heeft een andere afmeting dan de eerste.

Wat je wilt doen is wat we ook in andere e2e tests doen is de naam en een toevoeging meegeven, waarbij de toevoeging de scrrenshot uniek maakt.

Zoiets: 

`${Cypress.currentTest.title} - large`